### PR TITLE
First implementation of the GeoMat API

### PR DESCRIPTION
**Pull request by chgad**
First commit , just adding the files to the local requirements .txt file .

### CODE REVIEW

File: config/settings/local.py [hunk 1]
```diff
@@ -70,16 +70,26 @@
 # Additional local apps
 # ------------------------------------------------------------------------------
 INSTALLED_APPS += ('django_extensions',
-                   'rosetta', )
+                   'rosetta',
+                   'stdimage_serializer')
```
Comment (mimischi) [inline config/settings/local.py]: This _needs_ to go into `config/settings/base.py`!

File: requirements/local.txt [hunk 2]
```diff
@@ -26,3 +26,4 @@ django-rosetta==0.7.13
 djangorestframework==3.6.3
 markdown==2.1.0
 django-filter==1.0.1
+django-stdimage-serializer
```
Comment (mimischi) [inline requirements/local.txt]: This _needs_ to go into `requirements/base.txt`!

**Comment by mimischi** [comment]
I haven't fully checked this PR yet, but it would be nice to squash some commits, because they are either just [reverting stuff](https://github.com/GeoMatDigital/django-geomat/pull/55/commits/dda0c5164fb81dce920bbe6f79099563edfa8250) or [are empty](https://github.com/GeoMatDigital/django-geomat/pull/55/commits/0b0e5bbf00bc5cb0886a8c9ac2e1a695299097d7).

**Comment by chgad** [comment]
This was a bad attempt which got fixed a few weeks ago.

**Pull request closed without merging**